<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Temperatura</class>
 <widget class="QDialog" name="Temperatura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>temp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/recursos/temperatura 2.png</normaloff>:/recursos/temperatura 2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(145, 145, 145);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 33, 33);</string>
     </property>
     <property name="title">
      <string>Cent√≠grados</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdCent">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 0);</string>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialCent">
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(67, 67, 67);</string>
        </property>
        <property name="minimum">
         <number>-18</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 33, 33);</string>
     </property>
     <property name="title">
      <string>Fahrenheit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="lcdFahr">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 0);</string>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialFahr">
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(67, 67, 67);</string>
        </property>
        <property name="maximum">
         <number>210</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 33, 33);</string>
     </property>
     <property name="title">
      <string>Kelvin</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLCDNumber" name="lcdKel">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialKel">
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(67, 67, 67);</string>
        </property>
        <property name="minimum">
         <number>255</number>
        </property>
        <property name="maximum">
         <number>373</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
